CONTRIBUTING.md

```diff
@@ -110,6 +110,23 @@ git push --force-with-lease 25-annotations_to_tasks
 
 At this point, you're ready to make your changes! Feel free to ask for help. Everyone is a beginner at first :smile_cat:
 
+### Update an Example in Live IDE
+To update an example in a live IDE, follow these steps if you need to make significant changes to the example (not just updating the `bpmn-visualization` version):
```

Comment:
**question:** in the release documentation (stored in the bpmn-visualization repository), we mention to update the Live IDE example on major changes in the lib.
Do you suggest to do a fork in this case as well? If so, we should reference this paragraph in the release documentation.
We may move the part of the release documentation related to the examples repository here.

Reply:
:information\_source: As stated in the documentation, forking is required when there are modifications to be made to the example that are unrelated to the `bpmn-visualization` update.   
  
Consequently, I believe that this is not related to the release process because typically, example modifications are carried out during a sprint and not in the midst of a release.

Reply:
I was thinking about this step in the release process: https://github.com/process-analytics/bpmn-visualization-js/blob/v0.33.1/docs/contributors/maintainers.md#update-examples-running-in-live-ide.

The paragraph you add is clear enough, I didn't notice the `(not just updating the `bpmn-visualization` version)` mention.
So everything is fine.